### PR TITLE
chore(docs): fix imports

### PR DESCRIPTION
**Pull request by Shinigami92**
I found some old import statements and replaced them in the docs

### CODE REVIEW

**Review by damienwebdev — APPROVED** [review]
lgtm

**Comment by Shinigami92** [comment]
You already deleted the `doc` folder
This is not visible in the current PR, so please change the GitHub option to

image

File: doc/index.html [hunk 1]
```diff
@@ -85,8 +82,11 @@
 
 
 
-	
-	
+
+
+
+
+
```
Comment (import-brain) [inline doc/index.html:89]: I might not be seeing the full picture here, but what is this change for?

Comment (Shinigami92) [reply doc/index.html:89]: You can ignore it, we plan to delete old doc (not docs) folder anyway in another PR
This was just auto-prettier on save

Comment (import-brain) [reply doc/index.html:89]: > You can ignore it, we plan to delete old doc (not docs) folder anyway in another PR This was just auto-prettier on save

Ah, I see

**Review by import-brain — APPROVED** [review]

**Review by clarkerican — APPROVED** [review]

**Pull request merged**